BlueDBOnDisk/src/main/java/io/bluedb/disk/collection/BlueCollectionOnDisk.java

```diff
@@ -184,7 +183,6 @@ public void shutdown() {
 		recoveryManager.getChangeHistoryCleaner().stop();
 		rollupScheduler.forceScheduleRollups();
 		rollupScheduler.stop();
-		executor.shutdown();

```

Comment:
Hmm... its probably good practice to add a shutdown method on CachedWingleThreadingPool that calls shutdown on the cachedThreadPool field. That way we can be sure that we've communicated to the pool that we're shutting down. Wouldn't want threads waiting to timeout to delay the shutdown of the process.